### ISSUE TITLE
CSGPolygon following a path throws errors

### ISSUE THREAD

**Issue by DavidPgl** [godotengine/godot#70071]
### Godot version

4.0.beta8

### System information

Windows 10

### Issue description

CSGPolygon using the path mode will throw this error three times:
```
E 0:00:43:0492   looking_at: The target vector can't be zero.
  <C++ Error>    Condition "p_target.is_zero_approx()" is true. Returning: Basis()
  <C++ Source>   core/math/basis.cpp:1041 @ looking_at()
```

Tried it with instantiating a scene as well as creating everything in code:

![image](https://user-images.githubusercontent.com/35039404/207700649-9b137b50-da23-4ea2-b0cc-2a1113006981.png)

Not setting the `path_node` in the scene or in code prevents the errors but as this also stops the polygon from being displayed it's probably rendering related.

### Steps to reproduce

1. Add a `Path3D` and a `CSGPolygon3D` to the scene
2. Switch the `mode` inside the `CSGPolygon` to "Path"
3. Set the `path_node` inside the `CSGPolygon` to the `Path3D` node
4. Add some points to the `Path3D` curve
5. Start the scene

### Minimal reproduction project

[CSGPolygon3D_path_error.zip](https://github.com/godotengine/godot/files/10231174/CSGPolygon3D_path_error.zip)

**Comment by TokageItLab** [comment]
May be closed by #69726

**Comment by DavidPgl** [comment]
Seems to be fixed with 4.0.beta9
